example/ck_tile/17_grouped_gemm/quant_grouped_gemm.cpp

```diff
@@ -75,7 +89,7 @@ float grouped_gemm_tileloop(const ck_tile::stream_config& s,
                                                AccDataType,
                                                GemmShape,
                                                GemmUniversalTraits,
-                                               128>, // QuantGroupSize
```

Comment:
this argument must be a typename which has kM kN and kK as static integer members